api/Document.json

```diff
@@ -8173,12 +8169,12 @@
               "notes": "This event handler attribute is not supported; however, the event itself is supported since IE 10. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
             },
             "opera": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
+              "version_added": "49",
+              "notes": "Before Opera 49, this event handler attribute was not supported; however, the event itself was supported since Opera 20. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
             },
             "opera_android": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
+              "version_added": "46",
+              "notes": "Before Opera Android 46, this event handler attribute was not supported; however, the event itself was supported since Opera 20. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

Comment:
```suggestion
              "notes": "Before Opera Android 46, this event handler attribute was not supported; however, the event itself was supported since Opera Android 20. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```